Formula/stlink.rb

```diff
@@ -15,15 +15,13 @@ class Stlink < Formula
   end
 
   depends_on "libusb"
+  depends_on "gtk+3" => :optional
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
+  depends_on "cmake" => :build
```

Comment:
There's no way in hell we're adding `revision 0` to every formula in Homebrew 😜. If people want to harmonise the style on single `head` lines even further I'd settle on there always being an empty line between the initial block and `head`.